frontend/src/metabase/query_builder/containers/QueryBuilder.unit.spec.tsx

```diff
@@ -199,20 +200,15 @@ describe("QueryBuilder", () => {
         await screen.findByRole("button", { name: ".csv" }),
       );
 
-      expect(
-        mockDownloadEndpoint.called((url, options) => {
-          const { body: urlSearchParams } = options;
-          const query =
-            urlSearchParams instanceof URLSearchParams
-              ? JSON.parse(urlSearchParams.get("query") ?? "{}")
-              : {};
-
-          return (
-            url.includes("/api/dataset/csv") &&
-            query?.native.query === "SELECT 1"
-          );
-        }),
-      ).toBe(true);
+      const [url, options] = mockDownloadEndpoint.lastCall() as MockCall;
```

Comment:
Since we changed to use `GET` and `POST`. `options.body` turns out to be a `Promise` instead. I'm not totally sure why.